services/proctoring-service/src/controllers/proctoring.controller.js

```
@@ -72,9 +72,81 @@ async function getStudentsInExam(req, res) {
   }
 }
 
+/**
+ * Controller để bắt đầu một phiên thi
+ */
+async function startProctoringSession(req, res) {
+  try {
+    const userId = req.user.id; // Lấy userId từ token JWT
+    const { examId } = req.body;
+   // Thành dòng này:
```

Biome (2.1.2)

[error] Shouldn't redeclare 'startProctoringSession'. Consider to delete it or rename it.

'startProctoringSession' is defined here:


(lint/suspicious/noRedeclare)